packages/tldraw/src/lib/utils/export/export.ts

```diff
@@ -19,17 +19,17 @@ export async function getSvgAsImage(
 	svgString: string,
 	options: {
 		type: 'png' | 'jpeg' | 'webp'
-		quality: number
-		scale: number
 		width: number
 		height: number
+		quality?: number
+		pixelRatio?: number
 	}
 ) {
-	const { type, quality, scale, width, height } = options
```

Comment:
the use of scale in these options was confusing, as it's different from the scale that's in `TLImageExportOptions`